### PR TITLE
XW-4160 | Player migration script

### PR DESCRIPTION
**Pull request by hakubo**
I have a script to load the CSV but it contains secrets so I am not commiting it to repo

how to run?
SERVER_ID=177 php migrateOoyalatoJWPlayer.php

uncomment

### CODE REVIEW

File: maintenance/wikia/jwplayermigration/migrateOoyalatoJWPlayer.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+					$this->output( ' updated ' . $config['videoId'] . ' to ' . $this->map[$config['videoId']] . "\n" );
+					$config['mediaId'] = $this->map[$config['videoId']];
+					$config['player'] = 'jwplayer';
+				}
+			}
+
+
+			try {
+				//uncomment to do update
+				// WikiFactory::setVarById($varId,  $wikiId, $videoConfig , 'Automatic migration of Ooyala videos to JWPlayer');
```
Comment (hakubo) [inline maintenance/wikia/jwplayermigration/migrateOoyalatoJWPlayer.php]: uncomment to do the update

Comment (WikiaTech) [inline maintenance/wikia/jwplayermigration/migrateOoyalatoJWPlayer.php]: MAJOR Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: maintenance/wikia/jwplayermigration/migrateOoyalatoJWPlayer.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+		$this->map = $map;
+	}
+
+
+	public function execute() {
+		$this->dryRun = $this->hasOption( 'dry-run' );
+		$this->output( "Start \n" );
+
+		//$this->doUpdate( 1689, 'wgArticleVideoFeaturedVideos' );
```
Comment (WikiaTech) [inline maintenance/wikia/jwplayermigration/migrateOoyalatoJWPlayer.php]: MAJOR Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)

[project-id]: wikia:app

File: maintenance/wikia/jwplayermigration/migrateOoyalatoJWPlayer.php [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+					$this->output( ' updated ' . $config['videoId'] . ' to ' . $this->map[$config['videoId']] . "\n" );
+					$config['mediaId'] = $this->map[$config['videoId']];
+					$config['player'] = 'jwplayer';
+				}
+			}
+
+
+			try {
+				//uncomment to do update
+				// WikiFactory::setVarById($varId,  $wikiId, $videoConfig , 'Automatic migration of Ooyala videos to JWPlayer');
```
Comment (WikiaTech) [inline maintenance/wikia/jwplayermigration/migrateOoyalatoJWPlayer.php]: MAJOR Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)

[project-id]: wikia:app

File: maintenance/wikia/jwplayermigration/migrateOoyalatoJWPlayer.php [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+	public function execute() {
+		$this->dryRun = $this->hasOption( 'dry-run' );
+		$this->output( "Start \n" );
+
+		//$this->doUpdate( 1689, 'wgArticleVideoFeaturedVideos' );
+		//$this->doUpdate( 1749, 'wgArticleVideoFeaturedVideos2' );
+	}
+
+	private function doUpdate( $varId, $varName ) {
```
Comment (WikiaTech) [inline maintenance/wikia/jwplayermigration/migrateOoyalatoJWPlayer.php]: MAJOR Remove this unused private "doUpdate" method. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1144)

[project-id]: wikia:app

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 3 major

Watch the comments in this conversation to review them.

[project-id]: wikia:app
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request closed without merging**